go.mod

```diff
@@ -19,17 +18,16 @@ require (
 	github.com/nats-io/nats-streaming-server v0.17.0
 	github.com/nats-io/stan.go v0.6.0
 	github.com/networkservicemesh/api v0.0.0-20210202152048-ec956057eb3a
+	github.com/networkservicemesh/sdk-vpp v0.0.0-20210202164004-1ec8a9bd6287
```

Comment:
This looks like an artifact after rebase.
I guess here can be checkout go.mod, go.sum changes